xml_inscriptions/transcribed/NJ.Princ.PU.AM.L.y9.xml

```diff
@@ -126,55 +92,25 @@
       </surface>
-      <surface>
-         <!--Caption in desc element-->
-         <desc/>
-         <!--URL will actually only have the file name, which is the machine readable USEP number followed by ".jpg".-->
-         <graphic url="NJ.Princ.PU.AM.L.y9-Det.1.jpg"/>
-      </surface>
-      <surface>
-         <!--Caption in desc element-->
-         <desc/>
-         <!--URL will actually only have the file name, which is the machine readable USEP number followed by ".jpg".-->
-         <graphic url="NJ.Princ.PU.AM.L.y9-Det.2.jpg"/>
-      </surface>
-      <surface>
-         <!--Caption in desc element-->
-         <desc/>
-         <!--URL will actually only have the file name, which is the machine readable USEP number followed by ".jpg".-->
-         <graphic url="NJ.Princ.PU.AM.L.y9-Det.3.jpg"/>
-      </surface>
+      
    </facsimile>
    <text>
       
         <body>
         <div type="edition">
-            <ab>
-               <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <expan><abbr>M</abbr><ex>anibus</ex></expan>.
-               <lb n="2"/><persName><name type="gentilicium" key="Laberia">Laberiae</name> <name type="cognomen" key="Alexandria">Alexan
-               <lb n="3" break="no"/>driae</name></persName>, quae <supplied reason="omitted">vixit</supplied> <expan><abbr>ann</abbr><ex>is</ex></expan> <num value="10">X</num>,
-               <lb n="4"/><expan><abbr>m</abbr><ex>ensibus</ex></expan> <num value="5">V</num>, <expan><abbr>d</abbr><ex>iebus</ex></expan> <num value="7">VII</num>; et <persName><name type="cognomen" key="Silvanus">Silvano</name></persName>
-               <lb n="5"/><expan><abbr>q</abbr><ex>ui</ex></expan> <supplied reason="omitted">vixit</supplied> <expan><abbr>an</abbr><ex>nis</ex></expan> <num value="6">VI</num>, <expan><abbr>m</abbr><ex>ensibus</ex></expan> <num value="5">V</num>, <expan><abbr>d</abbr><ex>iebus</ex></expan> <num value="14">XIIII&gt;</num>.
-               <lg met="hexameter"><l><lb n="6"/>Hoc in sarcophago
-               <lb n="7"/>genetrix duo corpora
-               <lb n="8"/>natos</l> <l>condidit infelix
-               <lb n="9"/><expan><abbr>semperq</abbr><ex>ue</ex></expan> habitura dolorem</l>
-               <lb n="10"/><l>quo superat vivet <expan><abbr>vita<choice><corr>m</corr><sic>n</sic></choice>q</abbr><ex>ue</ex></expan>
-               <lb n="11"/>miser prima ducet</l> <l>coniuge
-               <lb n="12"/> praerepto genitor quibus ipse
-               <lb n="13"/>misellis.</l></lg>
-            </ab>
+            
          </div>
         
         
-                <div type="translation">
-                    <p/>
-                </div>
-            
+              
         
         
                 <div type="commentary">
-                    <p/>
+                    <p>This inscription was transferred to the Mead Museum Art Museum at Amherst College in 2012. Its
+               full edition is now <ref type="external"
```

Comment:
I'll double-check this, but this might need to be an `<a>` rather than a `<ref>`